### ISSUE TITLE
pin matplotlib version?

### ISSUE THREAD

**Issue by azazellochg** [MPI-Dortmund/napari-boxmanager#74]
The box manager fails with the latest matplotlib 3.9.0:
```
  File "/home/gsharov/soft/miniconda3/envs/napari-0.4.17/lib/python3.10/site-packages/box_manager/io/tloc.py", line 113, in to_napari
    colors = mcm.get_cmap("gist_rainbow")
AttributeError: module 'matplotlib.cm' has no attribute 'get_cmap'
```

Cause: https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.9.0.html#top-level-cmap-registration-and-access-functions-in-mpl-cm

Suggestion: pin matplotlib version to 3.8.4 in conda_env.yml

**Comment by thorstenwagner** [comment]
Hi!

Thanks for letting me know. I would rather prefer to fix the problem by

```python
    try:
        colors = mcm.get_cmap("gist_rainbow")
    except AttributeError:
        # for matplotlib > 3.9
        colors = matplotlib.colormaps["gist_rainbow"]
```

I tested it and it works :-)

**Comment by thorstenwagner** [comment]
Fixed with version 0.4.12! Thanks again!
